src/main/kotlin/no/nav/familie/ef/sak/fagsak/FagsakPersonController.kt

```diff
@@ -48,8 +56,22 @@ class FagsakPersonController(
                 person.id,
                 overgangsstønad = fagsaker.overgangsstønad?.let { fagsakService.fagsakTilDto(it) },
                 barnetilsyn = fagsaker.barnetilsyn?.let { fagsakService.fagsakTilDto(it) },
-                skolepenger = fagsaker.skolepenger?.let { fagsakService.fagsakTilDto(it) }
-            )
+                skolepenger = fagsaker.skolepenger?.let { fagsakService.fagsakTilDto(it) },
+            ),
+        )
+    }
+
+    @PostMapping("", "/person")
```

Comment:
Litt usikker på om vi trenger `/person` i endepunktet, men frontend bruker iaf kun `/person` 👀 
```suggestion
    @PostMapping("/person")
```

Reply:
Fjerner begge, jeg.